ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Header, Segment } from 'semantic-ui-react'
+import styled from 'styled-components'
+
+import { FILE_FIELD_NAME, PROJECT_DESC_FIELD, GENOME_VERSION_FIELD, FAMILY_FIELD_ID, INDIVIDUAL_FIELD_ID } from 'shared/utils/constants'
+import { BaseBulkContent, BASE_UPLOAD_FORMATS } from 'pages/Project/components/edit-families-and-individuals/BulkEditForm'
+import { INDIVIDUAL_CORE_EXPORT_DATA, INDIVIDUAL_ID_EXPORT_DATA } from 'pages/Project/constants'
+import ReduxFormWrapper from 'shared/components/form/ReduxFormWrapper'
+import { BooleanCheckbox } from 'shared/components/form/Inputs'
+import { VerticalSpacer } from 'shared/components/Spacers'
+
+const StyledLabel = styled.label`
+  color: rgba(0,0,0,.87);
+  font-size: .92857143em;
+  font-weight: 700;
+`
+
+const FIELD_DESCRIPTIONS = {
+  [FAMILY_FIELD_ID]: 'Family ID',
+  [INDIVIDUAL_FIELD_ID]: 'Individual ID (needs to match the VCF ids)',
+}
+const REQUIRED_FIELDS = INDIVIDUAL_ID_EXPORT_DATA.map(config => (
+  { ...config, description: FIELD_DESCRIPTIONS[config.field] }))
+
+const BLANK_EXPORT = {
+  filename: 'individuals_template',
+  rawData: [],
+  headers: [...INDIVIDUAL_ID_EXPORT_DATA, ...INDIVIDUAL_CORE_EXPORT_DATA].map(config => config.header),
+  processRow: val => val,
+}
+
+const UploadPedigreeField = React.memo(() =>
+  <div>
+    {/* eslint-disable-next-line jsx-a11y/label-has-for */}
+    <StyledLabel>Upload Pedigree Data</StyledLabel>
+    <Segment>
+      <BaseBulkContent
+        blankExportConfig={BLANK_EXPORT}
+        requiredFields={REQUIRED_FIELDS}
+        optionalFields={INDIVIDUAL_CORE_EXPORT_DATA}
+        uploadFormats={BASE_UPLOAD_FORMATS}
+        actionDescription="load individual data from an AnVIL workspace to a new seqr project"
+        url="/api/upload_temp_file"
+      />
+    </Segment>
+    <VerticalSpacer height={1} />
```

Comment:
I'm not sure if there any other better way to add spacing between these two fields.

Reply:
fields should always have a 10px margin under them by default so not entirely sure whats happening here. See if the change above fixes it

Reply:
also, if you are having trouble getting things to look right, feel free to set up a meeting with me and we can debug the styling issues together